### PR TITLE
Presentation proposal : Devops with Tuleap

### PR DESCRIPTION
**Pull request by vladomit**

### CODE REVIEW

**Comment by khesoem** [comment]
Hi @vladomitrovic 

Your presentation outline contains too much material for a short presentation. Be more specific, focus on a one or two items in your proposal.

Also, remember that your folder name should be `vladom-johanmb` not `vladom_johanmb`.

File: contributions/presentation/week5/vladom-johanmb/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ Johan Mallo Bakken (johanmb@kth.se)
+ Github username: [johanmallobakken](https://github.com/johanmallobakken)
+ 
+ ## Topic
+ 
+The goal is to present different agile methodologies approaches (scrum, kanban, scrumban, ...) and how they are related to Devops.
+
+- Introduction to agile methods
+- Scrum vs Kanban vs Scrumban
+- Tools examples (JIRA, Azure DevOps Boards, Tuleap, ...)
```
Comment (khesoem) [inline contributions/presentation/week5/vladom-johanmb/README.md]: Do you want to explain how these tools are used to implement agile methodologies or are these tools a whole different topic?

Comment (khesoem) [reply contributions/presentation/week5/vladom-johanmb/README.md]: Please make this point clear in your proposal.

**Review by khesoem — CHANGES_REQUESTED** [review]

File: contributions/presentation/week5/vladom-johanmb/README.md (hunk 1 above)
Comment (vladomit) [reply contributions/presentation/week5/vladom-johanmb/README.md]: Done

**Comment by khesoem** [comment]
I just noticed that we had a similar topic last year:
https://github.com/KTH/devops-course/pull/503

Can you make clear the differences between #503 and your proposal?

**Comment by vladomit** [comment]
@khaes-kth 
Hej, we changed our proposal. We want to focus on Tuleap.

**Comment by khesoem** [comment]
Good luck

**Pull request merged**